src/API/queryKeys.js

```diff
@@ -1,4 +1,5 @@
-export const SOURCES_QUERY_KEY = 'sources';
+export const sourcesQueryKey = (provider) => ['sources', provider];
+export const sourceUploadInfoKey = (source_id) => ['source_upload_info', source_id];
```

Comment:
Could you please use the key directly and skip this helper function wrapper? these helper functions create a new array on every re-render.

The reason I stoped using it is also because `useQuery` changed the queries  keys, now arrays are mandatory for performence wise